internal/DateTime/DateTime.js

```diff
@@ -18,9 +18,7 @@ const DateTimeBase = kind({
 	name: 'DateTime',
 
 	propTypes:  /** @lends agate/internal/DateTime.DateTime.prototype */ {
-		css: PropTypes.object,
-		locale: PropTypes.string,
-		value: PropTypes.any
+		css: PropTypes.object
```

Comment:
Would you please revert the change in this file? Since we added the prop and delete it from https://github.com/enactjs/agate/pull/357 intentionally. Without this, the DOM will have `locale` and `value` property that we don't want.